x-pack/plugins/security_solution/public/timelines/components/timeline/body/translations.ts

```diff
@@ -51,3 +51,10 @@ export const COLLAPSE = i18n.translate(
     defaultMessage: 'Collapse',
   }
 );
+
+export const ACTION_INVESTIGATE_IN_RESOLVER = i18n.translate(
+  'xpack.securitySolution.timeline.body.actions.investigateInResolverTooltip',
+  {
+    defaultMessage: 'Investigate in Resolver',
```

Comment:
we should give the new name of the resolver

Reply:
That's still TDB, so I noted in the `What's next?` section of this PR:

> - The action text `Investigate in Resolver` may be changed in a future PR